lib/modules/dosomething/dosomething_kudos/includes/Kudos.php

```diff
@@ -102,7 +119,7 @@ private function build($data, $full = false) {
       $northstar_response = json_decode($northstar_response);
 
       if ($northstar_response && !isset($northstar_response->error)) {
-        $northstar_user = array_shift($northstar_response->data);
+        $northstar_user = (array) $northstar_response->data;
```

Comment:
Would it make sense to use the [`$assoc` parameter of `json_decode`](http://php.net/manual/en/function.json-decode.php) here to avoid having to cast it?